EFCache.Redis.Tests/RedisCacheTests.cs

```diff
@@ -63,14 +64,16 @@ public void Item_not_returned_after_absolute_expiration_expired()
             var cache = new RedisCache(RegularConnectionString);
             var item = new TestObject { Message = "OK" };
 
-            cache.PutItem("key", item, new string[0], TimeSpan.MaxValue, DateTimeOffset.Now.AddMinutes(-10));
+            cache.PutItem("key", item, new string[0], TimeSpan.MaxValue, DateTimeOffset.Now.AddSeconds(1));
+
+            Thread.Sleep(1000);
 
             Assert.IsFalse(cache.GetItem("key", out var fromCache));
             Assert.IsNull(fromCache);
         }
 
         [TestMethod]
-        public void Item_not_returned_after_sliding_expiration_expired()
+        public void Cache_expires_expiration_expired()
```

Comment:
I didnt like this test method name - it is confusing and didnt mean anything. I tried to guess the intent from the test and come up with a better name: Item_not_returned_when_slidingexpiration_has_passed

Reply:
Actually, the previous test name is pretty similar!

Reply:
to be honest I don't remember why I changed the name and it does not make sense for me also :) 
Should I change it or you will since you have already merged it?